nncf/torch/model_transformer.py

```diff
@@ -139,7 +139,8 @@ def update_fused_bias(target_node_name: str, new_bias: Tensor, model: NNCFNetwor
     :param new_bias: New bias value.
     :param model: The model.
     """
-    fused_node = get_potential_fused_node(target_node_name, model)
+    nncf_graph = model.get_graph()
```

Comment:
Wouldn't it be very expensive to `get_graph` for every `update_fused_bias` call?

Reply:
No, it's nncfnetwrok contain `_compressed_graph` to save graph, and does not build graph on each call. https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/nncf_network.py#L425